instrumentation/opentelemetry-instrumentation-fastapi/tests/test_fastapi_instrumentation.py

```diff
@@ -124,6 +141,24 @@ def _create_app(self):
         self._instrumentor.instrument(tracer_provider=tracer_provider)
         return self._create_fastapi_app()
 
+    def _create_app_explicit_excluded_urls(self):
+        resource = Resource.create({"key1": "value1", "key2": "value2"})
+        tracer_provider, exporter = self.create_tracer_provider(
+            resource=resource
+        )
+        self.memory_exporter = exporter
+
+        to_exclude = "/user/123,/foobar"
+        excluded_urls = [
+            excluded_url.strip() for excluded_url in to_exclude.split(",")
+        ]
+        self._instrumentor.uninstrument()  # Disable previous instrumentation (setUp)
+        self._instrumentor.instrument(
+            tracer_provider=tracer_provider,
+            excluded_urls=to_exclude,
```

Comment:
```suggestion
            excluded_urls=excluded_urls,
```
Don't you mean this?

Reply:
Nope, that was a leftover from previous iterations. It expects a comma separated list of patterns. I will remove that variable as it is not used.